Library/Formula/gcovr.rb

```diff
@@ -0,0 +1,17 @@
+class Gcovr < Formula
+  homepage "http://gcovr.com/"
+  url "https://github.com/gcovr/gcovr/archive/3.2.tar.gz"
+  sha1 "97e44794969378db99cb77ee3c71b110dc634a8f"
+  head "https://github.com/gcovr/gcovr.git"
+
+  def install
+    prefix.install Dir["*"]
```

Comment:
Is this better going in ` libexec ` and then installing the symlink into ` bin ` or?..